CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Added `PortDriverPullDown` and `PortDriverPullUp` traits for port-expander ([#17]).
+- Added support for `PI4IOE5V6408` ([#17]).
```

Comment:
BTW, you don't need to add changelog entries.  I'll take care of that after merging PRs anyway.  And by leaving this up to me, you'll avoid having merge conflics in `CHANGELOG.md` all the time ;)